Ginger/GingerRuntime/GingerRuntime.csproj

```diff
@@ -40,7 +40,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Ginger.ExecuterService.Contracts" Version="4.3.1" />
```

Comment:
i will approve but why we have reference from GingerRuntime to the Contracts? should be only from GingerCoreNET no?